gprofiler/profilers/registry.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Dict, List, Optional, Type
+
+
+class ProfilerArgument:
```

Comment:
I see no need in this abstraction

Reply:
It's not an abstraction, but rather an improvement than the alternative of a dict

Reply:
It is an abstraction/wrapper over the arguments of `add_argument`. I agree that it looks better than a plain dict. But we don't need a class for it. A `namedtuple` will suffice, no?

Reply:
No, because I still need to get the dict without `None` values, and pretty much only a class can do that

Reply:
K. Can explain in comment then